examples/cockroachdb/client-secure.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 The Cockroach Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
```

Comment:
is it necessary to have this file here, is it not hosted somewhere.

Reply:
https://github.com/kanisterio/kanister/pull/1585#discussion_r939824949, included it after suggestion from @PrasadG193